meta-genivi-dev/recipes-sota/rvi/rvi_git.bbappend

```diff
@@ -0,0 +1,34 @@
+PR="r6"
+FILESEXTRAPATHS_prepend := "${THISDIR}:"
```

Comment:
FILESEXTRAPATHS_prepend := "${THISDIR}:" -> FILESEXTRAPATHS_prepend := "${THISDIR}:files" and move your toml and service files to "/files" directory